CodeChef_problems/Bob and His Friends/Solution.py

```diff
@@ -0,0 +1,44 @@
+
+
+'''
+Author : Shubhanshu Jain;
+'''
+
```

Comment:
#This is a simple problem
#Just take input of n, a, b, c and list in next line
#First we take Bob go to his room and change as minimum abs(a-b)+c
#Then we check from every room where bob can go and change